terraform/deployments/cloudfront/main.tf

```diff
@@ -301,6 +301,13 @@ resource "aws_wafv2_web_acl" "cdn_poc_govuk" {
     cloudwatch_metrics_enabled = true
     metric_name                = "cdn_poc_govuk"
   }
+
+  tags = {
```

Comment:
Please use [provider default tags](https://developer.hashicorp.com/terraform/tutorials/aws/aws-default-tags) instead of tagging every resource individually.